internal/cmd/connect/command_plugin_install.go

```diff
@@ -0,0 +1,456 @@
+package connect
+
+import (
+	"archive/zip"
+	"bytes"
+	"crypto/md5"
+	"crypto/sha1"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type manifest struct {
+	Owner struct {
+		Username string `json:"username"`
```

Comment:
Do we want to support any other formats? YAML? (or is JSON required here?)

Reply:
The manifests are JSON only, I believe.